src/bdev.rs

```diff
@@ -110,7 +112,27 @@ where
         }
     }
 
+    /// Returns claim module raw pointer.
+    fn claim_module_ptr(&self) -> *mut spdk_bdev_module {
+        unsafe {
+            let b = self.as_inner_ref().internal;
+            match b.claim_type {
+                SPDK_BDEV_CLAIM_NONE => std::ptr::null_mut(),
+                SPDK_BDEV_CLAIM_EXCL_WRITE => b.claim.v1.module,
+                _ => {
+                    let c = (*b.claim.v2.claims.tqh_first);
+                    assert!(
```

Comment:
Should this be a debug assert?

Reply:
I think no. It is better to panic on assert than to crash on random SEGFAULT.